tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -425,8 +425,9 @@ bool TRTEngineOp::ExecuteTrtEngine(OpKernelContext* ctx,
             const_cast<float*>(input_tensor.flat<float>().data());
         break;
       case nvinfer1::DataType::kHALF:
-        LOG(ERROR) << "FP16 inputs are not supported yet!";
-        return kRetry;
+        buffers[binding_index] =
```

Comment:
Thanks a lot for fixing this. Can we add a unit test for TRTEngineOp? We can mimic get_serialized_resource_op_test.cc and reuse the same OpTestBase utilities.